schematic_tables.txt

```diff
@@ -0,0 +1,2162 @@
+Minetest Game schematic tables
+==============================
+
+This document acts as the official record of all schematic tables used in
+Minetest Game, and should be updated by any commit that adds a table type
+schematic.
+
+The following tables are for pasting into mods that contain a function to
+convert the Lua tables into .mts files. Such mods often have two such functions
+which in this document are called 'mts_save()' and 'mts_save_up()':
```

Comment:
Instead of jumping in with function names, it would be nicer to explain the different ways a table can be structured so that it can be read by a mod. You can say "Minetest Game tables may be in the following formats:"... then explain them.

I'm not sure we should mention specific functions at all unless we direct people to a mod that uses them. (For example, in the mod I use mts_save() is what you call mts_save_up(), and unless we direct people somewhere they could end up using something that doesn't match this description.)